### ISSUE TITLE
Unable to create new Lab test template - AttributeError: 'LabTestTemplate' object has no attribute 'link_existing_item'

### ISSUE THREAD

**Issue by deepakc21** [earthians/marley#132]
Facing issue in creating new Lab test template in the "Healthcare" module.

AttributeError: 'LabTestTemplate' object has no attribute 'link_existing_item'
![tt3](https://user-images.githubusercontent.com/97949896/186339994-29c166e5-0629-42b4-b55a-3afee45fde15.jpg)

Details of apps installed on frappe cloud:
![tt4](https://user-images.githubusercontent.com/97949896/186340151-68a2540b-9575-4d10-8919-476ace1bff1f.jpg)

**Comment by ChillarAnand** [comment]
Please note that you are using V13 of frappe/erpnext. For v13, there is no need to install Frappe Health app. This will cause lot of conflicts.

Please use either
-  V13 of frappe/erpnext with "Healthcare" domain.
- V14 of frappe/erpnext + Frappe Health app.

Lab test template seems to be working fine on both v13 & v14.

Feel free to re-open the issue if you are able to reproduce it in either of the versions.

**Comment by deepakc21** [comment]
Hello Anand,

I have removed the Frappe Health App. 
![16](https://user-images.githubusercontent.com/97949896/196160202-27430dea-e731-4b97-83cf-de28ac821bb0.jpg)

I tried to create a "Lab test" and now we are having this error:
App healthcare is not installed
Failed to get method for command healthcare.healthcare.doctype.patient.patient.get_patient_detail with App healthcare is not installed.

![14](https://user-images.githubusercontent.com/97949896/196160406-f627f722-aa28-45c2-ac66-562dd8a2c6ab.jpg)

If I ignore this error and click on "Save" button then you will get another error:
Mandatory fields required in Lab Test
Gender
![15](https://user-images.githubusercontent.com/97949896/196160602-6d8ee288-fd75-40f5-8541-3acb4c03356d.jpg)

Gender field already has value in the Patient's data. But it is not being fetched in the Lab Test form (I have already raised a bug [here](https://github.com/frappe/health/issues/134))
